### PR TITLE
Added Query params to Transaction List and Session Delete API

### PR DESCRIPTION
**Pull request by mchopra-woolworths**

### CODE REVIEW

File: src/wpay-1.json [hunk 1]
```diff
@@ -2323,6 +2333,16 @@
               "default":1,
               "minimum":1
             }
+          },
+          {
+            "in":"query",
+            "name":"paymentSessionId",
+            "description":"f present, limits the list of transactions to those that relate to the payment session",
```
Comment (lohithv) [inline src/wpay-1.json:2340]: if ?

File: src/wpay-1.json [hunk 2]
```diff
@@ -185,6 +185,16 @@
...
```
Comment (lohithv) [inline src/wpay-1.json:192]: if

**Review by lohithv — APPROVED** [review]
LGTM

File: src/wpay-1.json (hunk 1 above)
Comment (mchopra-woolworths) [reply src/wpay-1.json]: typo. Fixed now.

File: src/wpay-1.json (hunk 2 above)
Comment (mchopra-woolworths) [reply src/wpay-1.json]: typo. Fixed now.

**Pull request merged**